src/wazuh_modules/wm_database.c

```diff
@@ -87,6 +93,23 @@ void wm_clean_dangling_groups();
 
 static void wm_sync_multi_groups(const char *dirname);
 
+/**
+ * @brief This method will read the legacy GROUPS_DIR folder to insert in the global.db the groups information it founds.
+ *        After every successful insertion, the legacy file is deleted. If we are in a worker, the files are deleted without inserting.
+ *        If the folder is empty, it will be removed.
+ */
+void wm_sync_legacy_groups_files();
+
+/**
+ * @brief Method to insert a single group file in the global.db. Like this insertion is performed for legacy group files only in the master,
```

Comment:
Ditto, we shouldn´t specify who or when this command will be called if it´s possible. It makes the description more generic for any use.

Reply:
Fixed !